examples/player/simid_player.html

```diff
@@ -50,11 +50,49 @@
       simidPlayer.initializeAd();
     }
 
+    /* Disables or enables stop, skip, and fatal error buttons
+     *  depending on if the ad is showing.
+     */
+    function setButtonState(disable) {
+      let adRequests = document.querySelectorAll(".ad_request");
+      adRequests.forEach((request) => {
+        request.disabled = disable;
+      });
+    }
+
     function playAd() {
       if (!simidPlayer) {
         initAd();
       }
       simidPlayer.playAd();
+      setButtonState(/* state= */ false);
```

Comment:
you changed the variable name to disable, so this should now be `/* disable= */`, same below

Reply:
fixed that